File: Practico8/ej6.pas
Program ej6;
Type
    Tnodo = ^Nodo;
    Nodo = record
        info: Integer;
        next: Tnodo;
    end;
Var
    x: Tnodo;
    y: Tnodo;
Begin { Main }
    New(x);
    New(y);
    y := x;
    Dispose(x);
End.

File: Practico8/ej6.c
#include <stdio.h>
#include <stdlib.h>

typedef struct Tnodo {
    int info;
    struct Tnodo *next;
} Tnodo;

int main(int argc, char **argv) {
    Tnodo *x = malloc(sizeof(Tnodo));
    Tnodo *y = x;
    free(x);
    printf("Valor de y: %d \n", y);

    return 0;
}
